app/views/topology/service.js

```diff
@@ -1359,7 +1359,7 @@ YUI.add('juju-topology-service', function(Y) {
       var name = service.displayName;
       if (service.pending) {
         // It will have parens added in the model.
-        name = name.replace(/^\(/, '').replace(/$\)/, '');
+        name = name.match(/^\(?(.{0,}?)\)?$/)[1];
```

Comment:
Any chance we can store this regex somewhere? If not, can we at least break it down in a comment somewhere and reference elsewhere?